tests/Psr3EveryLoggerTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Sil\Psr3Adapters\tests;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LogLevel as PsrLogLevel;
+use Sil\Psr3Adapters\Psr3ConsoleLogger;
+use Sil\Psr3Adapters\Psr3SamlLogger;
+use Sil\Psr3Adapters\Psr3SyslogLogger;
+use Sil\Psr3Adapters\Psr3Yii2Logger;
+
+class Psr3EveryLoggerTest extends TestCase
+{
+    public function testLogKnownLogLevel()
```

Comment:
This test seems to be doing too many things (since it is testing each of the different types of loggers).

If it is worth testing that providing the known log levels doesn't trigger exceptions, please create a separate test for each adapter, and they can all simply instantiate their adapter and pass it to a helper method that calls that adapter using each of the known log levels.

Reply:
Split out testing and extracted the test portion.